test/Microsoft.ML.Tests/Transformers/NormalizerTests.cs

```diff
@@ -410,10 +411,10 @@ public void SimpleConstructorsAndExtensions()
             var data4 = est4.Fit(data).Transform(data);
             var data5 = est5.Fit(data).Transform(data);
 
-            CheckSameSchemas(data1.Schema, data2.Schema);
-            CheckSameSchemas(data1.Schema, data3.Schema);
-            CheckSameSchemas(data1.Schema, data4.Schema);
-            CheckSameSchemas(data1.Schema, data5.Schema);
+            CommonUtilities.CheckSameSchemas(data1.Schema, data2.Schema);
```

Comment:
>CommonUtilities [](start = 12, length = 15)

Does CommonUtilities sounds a bit too generic as a name? How about TestCommon or TestUtilities? I would appreciate it if another reviewer can chime in on this on whether this needs to be changed or okay as is.

Reply:
Yes, this is good suggestion, will modify this

---
In reply to: [337126027](https://github.com/dotnet/machinelearning/pull/4346#discussion_r337126027) [](ancestors = 337126027)